staff/gianluca-bragaglia/app-template/contact.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact</title>
+</head>
+<body>
+    <header>
+        <h1>My App</h1>
+    </header>
+    <nav>
```

Comment:
You should move this ```<nav>```section into the bottom part of ```<header>```or the top part of ```<main>```, but do not leave it there floating.